config/stack/manifests/app.yaml

```diff
@@ -2,7 +2,7 @@ title: "Minimal GCP Environment"
 readme: ""
 overview: "This stack provisions a private network and creates resource classes that has minimal node settings and refer to that private network."
 overviewShort: "Start using GCP with Crossplane without needing to create your own resource classes!"
-version: 0.0.1
+version: 0.1
```

Comment:
This versioning had been discussed in a previous PR where I updated the ui-schema, I believe we wanted to leave this as `0.0.1` to match the other stacks and per semantic versioning spec <https://semver.org/>

Whenever versions should be updated, it will happen during the publishing pipeline when release branches are cut.

Reply:
Thanks for the heads-up! Just updated it.